pkg/server/server.go

```diff
@@ -251,11 +251,17 @@ func NewInstanceProvider(configFilePath, nodeName, internalIP, clusterDNS, clust
 	)
 
 	klog.V(5).Infof("setting up instance selector")
+	if instanceDataPath != "" {
```

Comment:
Do we need `override-instance-data` if setting `instance-data-path` to an empty string disables loading it anyway?

Reply:
good point, removed `override-instance-data` flag.